### ISSUE TITLE
[Bug]: [benchmark][cluster] query: mismatch FieldData in RetrieveResults

### ISSUE THREAD

**Issue by wangting0128** [milvus-io/milvus#11215]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: perf-20211102-8d0ad1d
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2): 2.0.0rc8.dev27
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

[logs-from-main-in-benchmark-jpbqm-1922291863.txt](https://github.com/milvus-io/milvus/files/7472187/logs-from-main-in-benchmark-jpbqm-1922291863.txt)

client log:
```
[2021-11-04 03:51:37,296] [   ERROR] - Error: <BaseException: (code=1, message=mismatch FieldData in RetrieveResults)> (pymilvus.client.grpc_handler:59)
[2021-11-04 03:51:37,297] [   ERROR] - Error: <BaseException: (code=1, message=mismatch FieldData in RetrieveResults)> (pymilvus.client.grpc_handler:59)
[2021-11-04 03:51:37,550] [   ERROR] - Error: <BaseException: (code=1, message=mismatch FieldData in RetrieveResults)> (pymilvus.client.grpc_handler:59)
[2021-11-04 03:51:39,296] [   DEBUG] - Milvus load_collection run in 9.1s (milvus_benchmark.client:52)
[2021-11-04 03:51:39,298] [   DEBUG] - Milvus load_collection run in 7.4s (milvus_benchmark.client:52)
```

server:
```
NAME                                                     READY   STATUS      RESTARTS   AGE     IP            NODE                                             NOMINATED NODE   READINESS GATES
benchmark-jpbqm-1-etcd-0                                 1/1     Running     0          3m37s   10.12.4.141   ip-10-12-4-121.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-etcd-1                                 1/1     Running     0          3m36s   10.12.0.205   ip-10-12-3-30.ap-southeast-1.compute.internal    <none>           <none>
benchmark-jpbqm-1-etcd-2                                 1/1     Running     0          3m36s   10.12.2.4     ip-10-12-0-102.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-datacoord-58ffd669cc-trns5      1/1     Running     0          3m36s   10.12.1.36    ip-10-12-0-243.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-datanode-7b845d7dfb-d5fn7       1/1     Running     0          3m36s   10.12.5.250   ip-10-12-4-49.ap-southeast-1.compute.internal    <none>           <none>
benchmark-jpbqm-1-milvus-datanode-7b845d7dfb-tz52b       1/1     Running     0          3m35s   10.12.2.204   ip-10-12-1-230.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-indexcoord-5d669544d8-phbm2     1/1     Running     0          3m36s   10.12.1.235   ip-10-12-2-163.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-indexnode-874898d9c-4dlqs       1/1     Running     0          3m36s   10.12.3.159   ip-10-12-2-163.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-indexnode-874898d9c-4jvng       1/1     Running     0          3m36s   10.12.6.205   ip-10-12-6-3.ap-southeast-1.compute.internal     <none>           <none>
benchmark-jpbqm-1-milvus-indexnode-874898d9c-k7pmw       1/1     Running     0          3m35s   10.12.6.11    ip-10-12-5-146.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-indexnode-874898d9c-q9tq8       1/1     Running     0          3m36s   10.12.2.6     ip-10-12-3-30.ap-southeast-1.compute.internal    <none>           <none>
benchmark-jpbqm-1-milvus-proxy-759f5d8c57-cf2l8          1/1     Running     0          3m35s   10.12.3.62    ip-10-12-1-230.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-querycoord-77b76df59-5l6zb      1/1     Running     0          3m36s   10.12.5.185   ip-10-12-5-146.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-querynode-58b4c45859-blmgj      1/1     Running     0          3m36s   10.12.0.135   ip-10-12-1-145.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-querynode-58b4c45859-c6k5g      1/1     Running     0          3m36s   10.12.4.212   ip-10-12-6-158.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-querynode-58b4c45859-dr5h9      1/1     Running     0          3m36s   10.12.0.171   ip-10-12-0-56.ap-southeast-1.compute.internal    <none>           <none>
benchmark-jpbqm-1-milvus-querynode-58b4c45859-q69f8      1/1     Running     0          3m36s   10.12.1.110   ip-10-12-0-243.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-querynode-58b4c45859-whknl      1/1     Running     0          3m36s   10.12.6.133   ip-10-12-7-155.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-querynode-58b4c45859-wrjkw      1/1     Running     0          3m36s   10.12.7.219   ip-10-12-4-121.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-milvus-rootcoord-7596879c49-54qsl      1/1     Running     0          3m36s   10.12.5.164   ip-10-12-7-155.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-minio-0                                1/1     Running     0          3m36s   10.12.1.171   ip-10-12-3-171.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-minio-1                                1/1     Running     0          3m36s   10.12.4.31    ip-10-12-5-146.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-minio-2                                1/1     Running     0          3m36s   10.12.1.205   ip-10-12-0-102.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-minio-3                                1/1     Running     0          3m35s   10.12.1.148   ip-10-12-0-229.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-pulsar-autorecovery-7bb7d5c6cd-9shc6   1/1     Running     0          3m36s   10.12.0.143   ip-10-12-0-229.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-pulsar-bastion-55bfdbf9dd-z5x75        1/1     Running     0          3m36s   10.12.5.8     ip-10-12-4-49.ap-southeast-1.compute.internal    <none>           <none>
benchmark-jpbqm-1-pulsar-bookkeeper-0                    1/1     Running     0          3m36s   10.12.6.224   ip-10-12-6-3.ap-southeast-1.compute.internal     <none>           <none>
benchmark-jpbqm-1-pulsar-bookkeeper-1                    1/1     Running     0          90s     10.12.1.61    ip-10-12-0-102.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-pulsar-broker-565689f88b-7nqd4         1/1     Running     0          3m36s   10.12.6.192   ip-10-12-4-49.ap-southeast-1.compute.internal    <none>           <none>
benchmark-jpbqm-1-pulsar-proxy-7855bb5746-c8qgk          2/2     Running     0          3m36s   10.12.5.65    ip-10-12-6-158.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-pulsar-zookeeper-0                     1/1     Running     0          3m36s   10.12.1.239   ip-10-12-0-102.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-pulsar-zookeeper-1                     1/1     Running     0          3m9s    10.12.3.61    ip-10-12-0-229.ap-southeast-1.compute.internal   <none>           <none>
benchmark-jpbqm-1-pulsar-zookeeper-2                     1/1     Running     0          2m43s   10.12.7.125   ip-10-12-6-77.ap-southeast-1.compute.internal    <none>           <none>
benchmark-jpbqm-1-pulsar-zookeeper-metadata-l2gmk        0/1     Completed   0          3m37s   10.12.3.98    ip-10-12-0-102.ap-southeast-1.compute.internal   <none>           <none>
```

### Steps To Reproduce

```markdown
argo task:benchmark-jpbqm

test yaml:
client-configmap:client-random-locust-search-84h-1b
server-configmap:server-cluster-8c64m-datanode2-indexnode4-querynode6-pulsar
```

**Comment by yanliang567** [comment]
@cydrain  this issue should be fixed in rc8, thanks

**Comment by wangting0128** [comment]
fix image： 2.0.0-rc8-20211104-cd29648
